xsensmt/XsensMT.cpp

```diff
@@ -125,99 +123,130 @@ bool XsensMT::open(yarp::os::Searchable &config)
 
     m_portInfo = XsPortInfo(comPortString, XsBaud::numericToRate(baudRate));
 
+	m_xsensControl = XsControl::construct();
```

Comment:
Pay attention to use 4-spaces and not tabs, thanks!

Reply:
Yes, it was the code copied from Xsens!

Fixed in https://github.com/robotology/yarp-device-xsensmt/pull/38/commits/a24afea392b3964a3b3a21fe8f5777d209cd88b6 and https://github.com/robotology/yarp-device-xsensmt/pull/38/commits/dbd12e7bb354cecee2db0a29948c6accecd19348

Reply:
Sorry for jumping in 😅 Can we fetch directly the code online?

Reply:
No, I was mentioned using the CMake. Instead of copy paste the library here is it possible to write a cmake machinery that fetches the code? (not related to the scope of this PR)

Reply:
No, there is no (official) public URL from which the code can be downloaded, if you go in their website you need to fill a form.